configs/edgenext/metafile.yml

```
@@ -0,0 +1,113 @@
+    Weights: https://download.openmmlab.com/mmclassification/v0/edgenext/edgenext- -7ca8a81d.pth
+    Config: configs/edgenext/edgenext-xxsmall_8xb256_in1k.py
+    Converted From:
+      Weights: https://github.com/mmaaz60/EdgeNeXt/releases/download/v1.0/edgenext_xxsmall.pth
+      Code: https://github.com/mmaaz60/EdgeNeXt
+  - Name: edgenext-xsmall_8xb256_in1k
+    Metadata:
+      FLOPs: 529970560
+      Parameters: 2336804
+    In Collections: EdgeNeXt
+    Results:
+      - Dataset: ImageNet-1k
+        Metrics:
+          Top 1 Accuracy: 74.86
+          Top 5 Accuracy: 92.31
+        Task: Image Classification
+    Weights: https://download.openmmlab.com/mmclassification/v0/edgenext/edgenext-xsmall_3rdparty_in1k_20220801-974f9fe7.pth
+    Config: configs/edgenext/edgenext-xsmall_8xb256_in1k.py
+    Converted From:
+      Weights: https://github.com/mmaaz60/EdgeNeXt/releases/download/v1.0/edgenext_xsmall.pth
+      Code: https://github.com/mmaaz60/EdgeNeXt
+  - Name: edgenext-small_8xb256_in1k
+    Metadata:
+      FLOPs: 1249788000
+      Parameters: 5586832
+    In Collections: EdgeNeXt
+    Results:
+      - Dataset: ImageNet-1k
+        Metrics:
+          Top 1 Accuracy: 79.41
+          Top 5 Accuracy: 94.53
+        Task: Image Classification
+    Weights: https://download.openmmlab.com/mmclassification/v0/edgenext/edgenext-small_3rdparty_in1k_20220801-d00db5f8.pth
+    Config: configs/edgenext/edgenext-small_8xb256_in1k.py
+    Converted From:
+      Weights: https://github.com/mmaaz60/EdgeNeXt/releases/download/v1.0/edgenext_small.pth
+      Code: https://github.com/mmaaz60/EdgeNeXt
+  - Name: edgenext-small-usi_8xb256_in1k
+    Metadata:
+      FLOPs: 1249788000
+      Parameters: 5586832
+    In Collections: EdgeNeXt
+    Results:
+      - Dataset: ImageNet-1k
+        Metrics:
+          Top 1 Accuracy: 81.06
+          Top 5 Accuracy: 95.34
+        Task: Image Classification
+    Weights: https://download.openmmlab.com/mmclassification/v0/edgenext/edgenext-small_3rdparty-usi_in1k_20220801-ae6d8dd3.pth
+    Config: configs/edgenext/edgenext-small_usi_8xb256_in1k.py
+    Converted From:
+      Weights: https://github.com/mmaaz60/EdgeNeXt/releases/download/v1.1/edgenext_small_usi.pth
+      Code: https://github.com/mmaaz60/EdgeNeXt
+  - Name: edgenext-base_8xb256_in1k
+    Metadata:
+      FLOPs: 3814395280
+      Parameters: 18511292
+    In Collections: EdgeNeXt
+    Results:
+      - Dataset: ImageNet-1k
+        Metrics:
+          Top 1 Accuracy: 82.48
+          Top 5 Accuracy: 96.2
+        Task: Image Classification
+    Weights: https://download.openmmlab.com/mmclassification/v0/edgenext/edgenext-base_3rdparty_in1k_20220801-9ade408b.pth
+    Config: configs/edgenext/edgenext-base_8xb256_in1k.py
+    Converted From:
+      Weights: https://github.com/mmaaz60/EdgeNeXt/releases/download/v1.2/edgenext_base.pth
+      Code: https://github.com/mmaaz60/EdgeNeXt
+  - Name: edgenext-base-usi_8xb256_in1k
```

Since the `usi` is only the training recipe, please place it in the training field.
```suggestion
  - Name: edgenext-base_3rdparty-usi_in1k
```

---

And please update the config file name at the same time.